src/groupeddataframe/grouping.jl

```diff
@@ -190,42 +248,76 @@ end
 
 """
     combine(gd::GroupedDataFrame)
-    combine(f::Function, gd::GroupedDataFrame)
+    combine(cols => f, gd::GroupedDataFrame)
```

Comment:
Why you do not want to provide the same API for `combine` as for `by`?

Actually using `groupby` and then `combine` is a common pattern IMHO. As you can perform grouping once, and then call `combine` many times later.

I can live with `map` not having the full API, as map is probably a bit less useful than `combine`, but we also could make them consistent.

If you agree with this approach we can decide to do it later as this will be non-breaking.

Reply:
I agree, I just wasn't sure about the inconsistency with `map`. I've changed `combine` to support the same options as `by`. Maybe we'll have to use another function in addition to `map`, e.g. `select` or `transform` (like in DataFramesMeta).